apps/e2e/src/client/accountSetup.ts

```diff
@@ -0,0 +1,50 @@
+import { createVertexClient, VertexClient } from '@vertex-protocol/client';
+import { addDecimals } from '@vertex-protocol/utils';
+import { runWithContext } from '../utils/runWithContext';
+import { RunContext } from '../utils/types';
+import { waitForTransaction } from '../utils/waitForTransaction';
+
+async function accountSetup(context: RunContext) {
+  const walletClient = context.getWalletClient();
+  const publicClient = context.publicClient;
+
+  const vertexClient: VertexClient = createVertexClient(context.env.chainEnv, {
+    walletClient,
+    publicClient,
+  });
+
+  const quoteProductId = 0;
+  const quoteMintAmount = addDecimals(1000, 6);
+  const quoteDepositAmount = addDecimals(500, 6);
```

Comment:
Was there a reason that we deposit less than we mint?

Reply:
Ah, is it for `quoteDepositWithReferralAmount`?

Should we make `collateralTests` runnable by itself? i.e. it mints, deposits, and withdraws without requiring this script